extensions/mongodb-client/runtime/pom.xml

```diff
@@ -47,13 +47,11 @@
             <groupId>org.graalvm.nativeimage</groupId>
             <artifactId>svm</artifactId>
         </dependency>
-
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-junit5-internal</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
```

Comment:
I don't think this is needed anymore, correct?

Reply:
Removed